saml-core-api/src/main/java/org/keycloak/saml/common/constants/GeneralConstants.java

```diff
@@ -98,6 +98,8 @@
 
     String SAML_RESPONSE_KEY = "SAMLResponse";
 
+    String SAML_ARTIFACT_KEY = "SAMLart";
```

Comment:
A bit of nitpick ... `SAMLArt` ?

Reply:
SAML specification defines the parameter as SAMLart. (http://docs.oasis-open.org/security/saml/v2.0/saml-bindings-2.0-os.pdf, Section 3.6.3.2)